platform-common/src/main/java/com/softicar/platform/common/core/i18n/IDisplayString.java

```diff
@@ -12,16 +12,14 @@
  * If a translatable text contains format specifiers (as in
  * {@link String#format(String, Object...)}), those are replaced with the
  * arguments given to the respective method.
- * <p>
- * FIXME i70184 should not implement {@link Comparable}
  *
  * @author Alexander Schmidt
  * @author Oliver Richers
  */
-public interface IDisplayString extends Comparable<IDisplayString> {
+public interface IDisplayString {
 
 	/**
-	 * An immutable, empty {@link IDisplayString}.
+	 * An empty {@link IDisplayString}.
 	 */
 	IDisplayString EMPTY = PlainDisplayString.getEmpty();
```

Comment:
IMO, there is not much use in this getEmpty() method, we could remove it
```suggestion
	IDisplayString EMPTY = new PlainDisplayString("");
```

Reply:
indeed